packages/matrix-client-server/src/index.test.ts

```diff
@@ -328,20 +328,20 @@ describe('Use configuration file', () => {
     })
     describe('/_matrix/client/v3/account/whoami', () => {
       let asToken: string
-      it('should reject if more than 100 requests are done in less than 10 seconds', async () => {
-        let token
-        let response
-        // eslint-disable-next-line @typescript-eslint/no-for-in-array, @typescript-eslint/no-unused-vars
-        for (const i in [...Array(101).keys()]) {
-          token = Number(i) % 2 === 0 ? `Bearer ${validToken}` : 'falsy_token'
-          response = await request(app)
-            .get('/_matrix/client/v3/account/whoami')
-            .set('Authorization', token)
-            .set('Accept', 'application/json')
-        }
-        expect((response as Response).statusCode).toEqual(429)
-        await new Promise((resolve) => setTimeout(resolve, 11000))
-      })
+      // it('should reject if more than 100 requests are done in less than 10 seconds', async () => {
+      //   let token
+      //   let response
+      //   // eslint-disable-next-line @typescript-eslint/no-for-in-array, @typescript-eslint/no-unused-vars
+      //   for (const i in [...Array(101).keys()]) {
+      //     token = Number(i) % 2 === 0 ? `Bearer ${validToken}` : 'falsy_token'
+      //     response = await request(app)
+      //       .get('/_matrix/client/v3/account/whoami')
+      //       .set('Authorization', token)
+      //       .set('Accept', 'application/json')
+      //   }
+      //   expect((response as Response).statusCode).toEqual(429)
+      //   await new Promise((resolve) => setTimeout(resolve, 11000))
+      // })
```

Comment:
Why comment this test ? Moved below ? Then drop

Reply:
Commented it to debug another test since it takes a lot of time to run, and forgot to uncomment it, thank you for noticing